server/repository/src/db_diesel/patient.rs

```diff
@@ -490,12 +480,9 @@ mod tests {
             )
             .unwrap();
         assert_eq!(result.len(), 0);
-        // test filter for name_or_code and identifier
         let result = repo
             .query_by_filter(
-                PatientFilter::new()
-                    .name_or_code(StringFilter::equal_to("codePatient"))
-                    .identifier(StringFilter::like("nhn")),
+                PatientFilter::new().identifier(StringFilter::like("test_name")),
```

Comment:
Nit: add test cases for `code`/`code2` or for an identifier matching two patient,e.g.

|Name| code |code2|
|----|--|--|
|Patient1| test1| code11 |
|Patient2| code22| test2 |

and then search for `test`